libs/ci/bitbucket/bitbucket.go

```
@@ -492,6 +517,137 @@ func (b BitbucketAPI) GetUserTeams(organisation string, user string) ([]string,
+	if err != nil {
+		fmt.Println("Error reading response:", err)
+		return "", fmt.Errorf("error reading response: %v", err)
+	}
+
+	var triggerPipelineResponse PipelineResponse
+	if err := json.Unmarshal(body, &triggerPipelineResponse); err != nil {
+		fmt.Println("Error parsing response:", err)
+		return "", fmt.Errorf("error parsing response: %v", err)
+	}
+
+	log.Printf(triggerPipelineResponse.Links.HTML.Href)
```

golangci-lint (1.62.2)

printf: non-constant format string in call to log.Printf

(govet)